### ISSUE TITLE
iozone sha256 mismatch on install Version 3.491

### ISSUE THREAD

**Issue by a7z** [Homebrew/homebrew-core#77494]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
$ brew config
HOMEBREW_VERSION: 3.1.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 92cf9047191393a2d624c64fc28d9595309ba273
Last commit: 7 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: d419775e7df0949c2e1d4facc437f52eef8b9f67
Core tap last commit: 4 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 2
Homebrew Ruby: 2.6.3 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: dual-core 64-bit penryn
Clang: 8.0.0 build 800
Git: 2.31.1 => /usr/local/bin/git
Curl: 7.43.0 => /usr/bin/curl
macOS: 10.11.6-x86_64
CLT: 8.2.0.0.1.1480973914
Xcode: N/A

$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You are using macOS 10.11.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
old version.
```

### 

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

installing iozone per "brew install iozone" to test the filesystem speed

### What happened (include all command output)?

$ brew install iozone
Updating Homebrew...
Warning: You are using macOS 10.11.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
old version.

==> Downloading https://www.iozone.org/src/current/iozone3_491.tgz
Already downloaded: /Users/arne/Library/Caches/Homebrew/downloads/1dc4c9f7eb7576acaf3a46eb2dd2ffd010dc398b7c48784b8fdc1e1496029c40--iozone3_491.tgz
Error: SHA256 mismatch
Expected: 057d310cc0c16fcb35ac6de25bee363d54503377cbd93a6122797f8277aab6f0
  Actual: 14ce89b60a43c21f1fadb82f15c1b926c993f3c6ad2a647202a6ccf117c72317
    File: /Users/arne/Library/Caches/Homebrew/downloads/1dc4c9f7eb7576acaf3a46eb2dd2ffd010dc398b7c48784b8fdc1e1496029c40--iozone3_491.tgz
To retry an incomplete download, remove the file above.

### What did you expect to happen?

That it installs normaly.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew install iozone
```

**Comment by gromgit** [comment]
The source tarball has indeed changed from the original: 4.91 was bottled on Dec 19 2020, but the current source tarball is dated Jan 12 2021.

I've dropped upstream an email query. Stay tuned...

**Comment by nandahkrishna** [comment]
Fixed in #77640.
